app/scripts/controllers/network-order.ts

```diff
@@ -163,10 +148,11 @@ export class NetworkOrderController extends BaseController<
    * @param networkList - The list of networks to update in the state.
    */
 
-  updateNetworksList(networkList: []) {
+  updateNetworksList(chainIds: [Hex]) {
```

Comment:
just making sure, `[Hex]` and not `Hex[]`

Reply:
Ah thanks it should be `Hex[]`

Reply:
https://github.com/MetaMask/metamask-extension/pull/26433/commits/ae608b290ad7e987c76cbdd37a3bde56dfd57d88